dm/tests/checkpoint_transaction/conf/dm-worker1.toml

```diff
@@ -1,2 +1,5 @@
-name = "worker1"
 join = "127.0.0.1:8261"
+keepalive-ttl = 1
+log-level = "info"
```

Comment:
in fact `log-level` and `worker-addr` are passed from command line arguments in tests